src/controller/python/BUILD.gn

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("${chip_root}/gn/chip/tools.gni")
+
+config("controller_wno_deprecate") {
+  cflags = [ "-Wno-deprecated-declarations" ]
+}
+
+config("controller_wno_warning_error") {
```

Comment:
Please don't disable -Werror. Only disable warnings.

The build should be silent.

Reply:
Done, thanks